packages/calcite-components/src/components/popover/popover.tsx

```diff
@@ -307,6 +307,11 @@ export class Popover
 
   componentDidLoad(): void {
     setComponentLoaded(this);
+    if (this.referenceElement && !this.effectiveReferenceElement) {
+      this.setUpReferenceElement();
+    }
+    connectFloatingUI(this, this.effectiveReferenceElement, this.el);
```

Comment:
` this.setUpReferenceElement();` already calls `connectFloatingUI` so no need here.

Reply:
Thanks for the catch! Forgot to carry that over from previous feedback.